### PR TITLE
fix: Do not import from `axios` but only `@nextcloud/axios`

### PR DESCRIPTION
**Pull request by susnux**
## Summary

Do not import from `axios` this is extranous, but only from `@nextcloud/axios`.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/utils/handlers.ts [hunk 1]
```diff
@@ -43,6 +44,7 @@ export const handleError = (error, message) => {
 		fullMessage += t('settings', 'There were too many requests from your network. Retry later or contact your administrator if this is an error.')
 	}
 
+	fullMessage = fullMessage || t('Settings', 'Error')
```
Comment (Pytal) [inline apps/settings/src/utils/handlers.ts:47]:
```suggestion
	fullMessage = fullMessage || t('settings', 'Error')
```

**Review by Pytal — APPROVED** [review]

**Comment by skjnldsv** [comment]
Can you explain 65973d009b6b80207ee95f9d8a35dab882cd3fe9 ? :)

**Comment by susnux** [comment]
> Can you explain https://github.com/nextcloud/server/commit/65973d009b6b80207ee95f9d8a35dab882cd3fe9 ? :)

See if CI gets green, because currently this is blocked (that commit is from https://github.com/nextcloud/server/pull/45207 ). Meaning we can not use latest `@nextcloud/axios` because webdav does not work with `X-Requested-With: XMLHttpRequest`.

CI looks good so dropped the commit again, but adjusted the target so we chain onto #45207

**Review by miaulalala — APPROVED** [review]

**Comment by susnux** [comment]
Still waiting for https://github.com/nextcloud/server/pull/45207

**Pull request merged**